feat: add params schema docs for includes (#311)
livingdocsIO/livingdocs: reference-docs/includes/server_customization.md

```js
// Example include service using the paramsSchema
module.exports = {
    name: 'youtubeIncludeService',
    paramsSchema: [
      {
        type: 'li-text',
        handle: 'url', // <-- 1. register url form field in sidebar
        config: {
          maxLength: 200
        },
        ui: {
          component: 'liMetaTextareaForm',
          config: {
            label: 'Youtube settings',
            placeholder: 'Paste Embed Code or URL'
          }
        }
      }
    ],
    defaultParams: {
      url: ''
    },
    rendering: {
      type: 'function',
      render: (params, options) => {
        const {url} = params  // <-- 2. use the params to render the include
        return `<div> do something with the url: ${url}</div>` 
      }
    }
  } 
}
```

#### More flexibility
The `uiComponents` array allows you to define a list of ui elements that are rendered in the sidebar upon selection of the `doc-include` (top to bottom). As you register your own component or helper provided by livingdocs, you have more choices but the development time increases as well. 

> You can choose between 3 types or  you can generate the sidebar.

Reviewer: ```suggestion
You can choose between 3 types of custom UI components.
```

Reviewer: I wouldn't mention the possibility for generation again here as this is specifically about the "more flexibility"
